### ISSUE TITLE
Training too long kill model. Is it overfitting?

### ISSUE THREAD

**Issue by likeoff** [minimaxir/textgenrnn#154]
Hello! I am going to find why after several epochs textgenrnn is unable to generate text.

I run code:

`textgen.train_from_file('Motherboards.txt', rnn_layers=2, rnn_size=256, new_model=0, batch_size=100, max_length=500, train_size=1, dropout=0, gen_epochs=0, num_epochs=15)`

Here at 6-th epoch dramatically decrease of loss and 

```
Training on 1,240,092 character sequences.
Epoch 1/10
12400/12400 [==============================] - 5369s 433ms/step - loss: 3.0435
Epoch 2/10
12400/12400 [==============================] - 5360s 432ms/step - loss: 1.7512
Epoch 3/10
12400/12400 [==============================] - 5341s 431ms/step - loss: 1.3421
Epoch 4/10
12400/12400 [==============================] - 5388s 435ms/step - loss: 1.2557
Epoch 5/10
12400/12400 [==============================] - 5420s 437ms/step - loss: 0.4390
Epoch 6/10
12400/12400 [==============================] - 5413s 437ms/step - loss: 1.1921e-07
Epoch 7/10
12400/12400 [==============================] - 5419s 437ms/step - loss: 1.1921e-07
Epoch 8/10
12400/12400 [==============================] - 5373s 433ms/step - loss: 1.1921e-07
Epoch 9/10
12400/12400 [==============================] - 5383s 434ms/step - loss: 1.1921e-07
Epoch 10/10
12400/12400 [==============================] - 5378s 434ms/step - loss: 1.1921e-07

```
when I generate text with prefix = 'intel'

`textgen.generate(prefix = 'intel', max_gen_length=1048, temperature=0.9, n=1 )`

it generates 

> intel

**Comment by breadbrowser** [comment]
why the fuck your so big 12.400 you only need 3,500-4,000 any more and you will get training loss
